src/app/workflow/launch-third-party/multi-cloud-launch/multi-cloud-launch.component.scss

```diff
@@ -0,0 +1,4 @@
+// Since this appears a small dropdown, font-size is decreased. Better matches Kim's mock
+.mat-form-field {
+  font-size: 12px;
```

Comment:
unless you're absolutely sure we won't have a small dropdown, put it in the global styles.scss and add an additional class identifier for it (so that all mat-form-fields with the "small" class for example would have its font size changed). hopefully we would remember to use it